kedro/framework/cli/starters.py

```diff
@@ -704,9 +704,14 @@ def _validate_config_file_against_prompts(
     if config is None:
         raise KedroCliError("Config file is empty.")
     missing_keys = set(prompts) - set(config)
-    if missing_keys:
+    if "project_name" in missing_keys:
```

Comment:
thanks @ankatiyar , great job!
Maybe we should make a slight change, like this:
missing_keys_without_mandratory = missing_keys - set(["example_pipeline", "tools"])
if missing_keys_without_mandratory:
        click.echo(yaml.dump(config, default_flow_style=False))
        raise KedroCliError(f"{', '.join(missing_keys)} not found in config file.")
This change will allow us to check not only the project_name but also all other prompts that users are waiting for (except for the mandatory example_pipeline and tools), similar to how it was previously.

Reply:
Updated!